.github/reviewers/mentor-leaderboard.md

```
@@ -1,96 +1,102 @@
-| #78 | @syedimtiyaz-1 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #79 | @tarunyaprogrammer | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #80 | @techrunnerbysj | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #81 | @thakurutkarsh22 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #82 | @the-matrixneo | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #83 | @topshe23 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #84 | @uddalak2005 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #85 | @ultrabot05 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #86 | @vanshaggarwal07 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #87 | @whyankush07 | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
-| #88 | @xthxr | 0.00 | 0 | 0 | 0.00 | 🔴 Inactive |
+### 📊 Scoring System
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)